src/app/modules/main/wallet_section/transactions/module.nim

```diff
@@ -10,12 +10,26 @@ import ../../../../../app_service/service/network/service as network_service
 
 export io_interface
 
+type TmpTransactionDetails = object
```

Comment:
nit: SendTransactionDetails, this is the info we use to create the transaction and not the transaction by himself

Reply:
Fixed, also added a short note that type should not be public ever, it's introduced like a local type to that module, just to bound and keep things simpler.